src/apis/us/UserCommImpl.java

```diff
@@ -0,0 +1,66 @@
+package apis.us;
+
+
+public class UserCommImpl implements UserCommHandler{
+
+    String outPutLocation=null;
+    String inputType=null;
+    //SEE THE UserCommHandler TO FIND OUT WHAT THESE DO!!!
+    
+    @Override
+    public void getUserInput() {
+        //Put a scanner here or something, idk
+        //I'm not sure how this even runs
+        
+        
+    }
+
+    @Override
+    public void setDelimiters() {
+        // Set the Delimiters for the output/input? Call setOutputType?
+        
+    }
+
+    @Override
+    public String setOutputType(String stringParam) {
```

Comment:
might be better to use enums rather than a string system

Reply:
Great Idea!
possible outputs
-Console
-CSV file
-Text file
-JSON